### PR TITLE
Fix border and spacing on rich text fields

### PR DESCRIPTION
**Pull request by mgwalker**
### This pull request changes...
- removes border around rich text element
- removes min-height on rich text

### CODE REVIEW

File: web/src/styles/scss/print.scss [hunk 1]
```diff
@@ -38,11 +38,10 @@
     display: none;
   }
 
-
   .ds-l-container,
   .ds-l-col--9 {
-    width: 100%!important;
-    min-width: 100%!important;
+    width: 100% !important;
+    min-width: 100% !important;
```
Comment (hbillings) [inline web/src/styles/scss/print.scss]: Nitpicking the CSS, I see. I approve. 😆

File: web/src/styles/scss/print.scss [hunk 2]
```diff
@@ -80,6 +79,15 @@
...
     border: none !important;
   }
+
+  .public-DraftEditor-content {
+    border: 0;
+  }
+
+  .rte-textarea-l .public-DraftEditor-content,
+  .textarea-l {
+    min-height: 0;
```
Comment (hbillings) [inline web/src/styles/scss/print.scss]: Was there a reason this had a `min-height` at all?

Comment (mgwalker) [reply web/src/styles/scss/print.scss]: No idea.  Maybe it's to mimic the textareas?  Without a `min-height`, it's just as tall as the content, so it kind of looks like a one-line textfield.

File: web/src/styles/scss/print.scss (hunk 1 above)
Comment (mgwalker) [reply web/src/styles/scss/print.scss]: Blame `prettier`!

**Pull request merged**